src/main/java/klerer/rijks/json/RijksCollection.java

```diff
@@ -0,0 +1,7 @@
+package klerer.rijks.json;
+
+public class RijksCollection {
+
+    public ArtObject [] rijksCollection;
```

Comment:
The names of variables in your classes must match the names of variables in the json